OpenShift/helm/nexus-iq/Chart.yaml

```diff
@@ -1,9 +1,25 @@
-apiVersion: v1
-appVersion: 1.84.0
-description: A Helm chart for Kubernetes
-name: iqserver
-version: 0.1.0
+apiVersion: v2
+name: nexus-iq-server
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+version: 90.0.0
```

Comment:
over on the operator side i had settled on a pattern of ${app-version}-${operator-build-number}. I might like to see such a thing here as well, so we can keep it in sync with the application it's deploying.  What's your motivation for the 90.0.0 pattern?

Reply:
I wanted it to better align to our versioning and I felt our major versions tend to be a new product (nxrm2 vs 3). This let me match us with the first two digit and increment the third for the chart

Reply:
your approach sounds better, I wasn't sure a -nn would work